eurostag-integration/src/main/java/eu/itesla_project/eurostag/EurostagExportTool.java

```diff
@@ -70,18 +67,29 @@ public void run(CommandLine line) throws Exception {
 
         System.out.println("exporting ech...");
         // export .ech and dictionary
-        EurostagEchExportConfig exportConfig = new EurostagEchExportConfig();
-        BranchParallelIndexes parallelIndexes = BranchParallelIndexes.build(network, exportConfig);
-        EurostagDictionary dictionary = EurostagDictionary.create(network, parallelIndexes, exportConfig);
-        new EurostagEchExport(network, exportConfig, parallelIndexes, dictionary).write(outputDir.resolve("sim.ech"));
+        EurostagEchExportConfig exportConfig = EurostagEchExportConfig.load();
+        EurostagFakeNodes fakeNodes = EurostagFakeNodes.build(network, exportConfig);
+        BranchParallelIndexes parallelIndexes = BranchParallelIndexes.build(network, exportConfig, fakeNodes);
+        EurostagDictionary dictionary = EurostagDictionary.create(network, parallelIndexes, exportConfig, fakeNodes);
 
         try (Writer writer = Files.newBufferedWriter(outputDir.resolve("sim.ech"), StandardCharsets.UTF_8)) {
             EsgGeneralParameters parameters = new EsgGeneralParameters();
             parameters.setTransformerVoltageControl(false);
             parameters.setSvcVoltageControl(false);
-            EsgNetwork networkEch = new EurostagEchExport(network, exportConfig, parallelIndexes, dictionary).createNetwork(parameters);
+            parameters.setMaxNumIteration(eurostagConfig.getLfMaxNumIteration());
+            EsgSpecialParameters specialParameters = null;
+            if (exportConfig.isSpecificCompatibility()) {
+                System.out.println("specificCompatibility=true: forces start mode to WARM and write the special parameters section in ech file");
```

Comment:
The run method is deprecated. Use the new run method instead and log output to the outputstream available through the ToolRunningContext instance.